core/table.ts

```diff
@@ -192,6 +199,13 @@ export interface ITableConfig extends ITargetableConfig, IDocumentableConfig, ID
    * If configured, relevant assertions will automatically be created and run as a dependency of this dataset.
    */
   assertions?: ITableAssertions;
+
+  /**
+   * Unique keys for merge criteria for incremental tables.
+   *
+   * If configured, records with matching unique key(s) will be updated, rather than new rows being inserted.
+   */
+  uniqueKeys?: string[];
```

Comment:
I'm sorry to do this to you, but I'm just realizing we already use `uniqueKey` for assertions. This kind of sucks if they are different. Maybe we should leave this as it was (sorry!!).

Reply:
That's ok, one global find and replace